frontend/i18next.scanner/es/translation.json

```diff
@@ -583,5 +583,12 @@
   "Show user menu": "Mostrar el menú de usuario",
   "Participate": "Participar",
   "My agendas": "Mis agendas",
-  "My profile": "Mi perfil"
+  "My profile": "Mi perfil",
+  "My Spaces": "Mis espacios",
+  "My Password": "Mi contraseña",
+  "My Configuration links": "Mis enlaces de configuración",
+  "Their Account": "Sus cuenta",
```

Comment:
In fact, this is an unusual formulation which seems not totally clear for me, whether it be in English or in Spanish.

The [neutral their](https://english.stackexchange.com/questions/48/is-there-a-correct-gender-neutral-singular-pronoun-his-vs-her-vs-their) is usually used to address a plural, which is not true in our case, as we are talking about one account (if I'm not mistaken).

Something like `Account of {username}` or `{username}'s account`  seems clearer to me, is it something that can be considered?

Reply:
Indeed this could be a solution, however in terms of UI, a non-generic wording could cause multiple problems, first of all this could break some of the interface responsiveness, and secondly, as a user I'd prefer a generic wording for menus, so the interface is always the same when editing a user, no difference on text, number of lines (wrapping for example)...